install.sh

```diff
@@ -9,6 +13,7 @@ needed_package_name=null_package_name
 stage=release
 interactive=1
 skip_install=0
+skip_checks=0
```

Comment:
no, I didn't mean skip any existing checks. my thought was that this new check isn't covering the existing checks (like api token validation).  and it doesn't make since for a possible/future check were the install would fail if not writable.  and it has value to control only this particular check.